cmd/databricks-exporter/main.go

```diff
@@ -34,14 +34,12 @@ import (
 )
 
 var (
```

Comment:
Potential: Add flags/config options for configurable variables, e.g. timeouts, window sizes, etc?